src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.InvalidDataForRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static StorageDao storageDao;
+    private static User anotherUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        anotherUser = new User();
+        anotherUser.setLogin("bobik");
+        anotherUser.setAge(30);
+        anotherUser.setPassword("idk1488");
+        user = new User();
+        user.setPassword("popbob1337");
+        user.setAge(20);
+        user.setLogin("bobcarlson");

```

Comment:
when you make your user non-static you can initialize it in @BeforeEach annotated method